src/Splitter.sol

```diff
@@ -60,6 +63,9 @@ contract Splitter {
         
         vat.hope(_daiJoin);
         
+        // Initial value for safety
+        hop  = 1 hours;
```

Comment:
neat - can put the comment in the same line.

Reply:
Changed in 79a0021c